http/src/main/resources/reference.conf

```diff
@@ -756,9 +756,9 @@ auth {
   providerClass = "org.broadinstitute.dsde.workbench.leonardo.auth.sam.SamAuthProvider"
   providerConfig {
     samServer = ${sam.server}
-    petTokenCacheEnabled = true
-    petTokenCacheExpiryTime = 60 minutes
-    petTokenCacheMaxSize = 1000
+    petKeyCacheEnabled = true
```

Comment:
I changed `HttpSamDAO` to cache pet _keys_ (which is what we're actually requesting from Sam) instead of pet tokens. We can generate tokens from keys on-demand. Keys don't expire, so this lets us have a longer cache expiry, which should improve performance I think.

Reply:
ah that's great!..thanks!